nntrainer/src/conv2d_layer.cpp

```diff
@@ -79,24 +79,25 @@ void Conv2DLayer::read(std::ifstream &file) { Layer::read(file); }
 
 void Conv2DLayer::save(std::ofstream &file) { Layer::save(file); }
 
-Tensor Conv2DLayer::forwarding(Tensor in, int &status) {
+sharedTensor Conv2DLayer::forwarding(sharedTensor in) {
+  int status = ML_ERROR_NONE;
+  input = in.get()[0];
```

Comment:
Minor though, I think in this context should be star. (and for other occurrences of get()[0])
```suggestion
  input = *in;
```

Reply:
yes, `*in` is more suitable for layers who only deal with a single input than `in.get()[0]`.
Done.